docs/sources/tempo/setup/upgrade.md

```diff
@@ -84,7 +84,11 @@ vParquet4 format is now the default block format.
 It's production ready and we highly recommend switching to it for improved query performance. [PR [3810](https://github.com/grafana/tempo/pull/3810)]
 
 Upgrading to Tempo 2.6 modifies the Parquet block format.
-Although you can use Tempo 2.6 with vParquet2 or vParquet3, you can only use Tempo 2.6 with vParquet3.
+You don't need to do anything with Parquet to go from 2.5 to 2.6.
```

Comment:
same comments as the release notes

Reply:
Thank you for reading this! I'm glad you read through this.